### ISSUE TITLE
Error in plugin 'sass'

### ISSUE THREAD

**Issue by Larmor27** [zurb/foundation-sites#8469]
## Software used
OS Windows 7
npm -v : 3.8.3
node -v : v4.4.1
bower -v : 1.7.7
node-sass -v: node-sass 3.4.2 (Wrapper) [JavaScript], libsass 3.3.2 (Sass Compiler) [C/C++]
git --version: 2.7.4.windows.1

## How to reproduce
I basically followed the documentation [here](http://foundation.zurb.com/sites/docs/installation.html)

1. First I run `npm install --global foundation-cli`, then `foundation new `using the Git CMD. I choose to build a website, call my project "test" and choose **(A)** the basic template option with the sass compiler included or **(B)** the ZURB Template. My machine runs a wamp environment, windows is installed on C: whereas the project is situated in E:\wamp\www\test
2. I run `foundation watch `in E:\wamp\www\test

**What did you expect to happen?**
I expected to have foundation watching my files, updating my website at http://localhost:8000 when I change the sass at assets\scss\_setting.scss

**What happened instead?**
 `foundation watch`...

**(A)** times out after finishing 'default'. I have to kill the process with ctr+C to proceed. The shell remains active during all this.

```
 E:\wamp\www\test>foundation watch
 
 foundation-sites-template@1.0.0 start E:\wamp\www\test
 gulp
 
 [19:09:05] Using gulpfile E:\wamp\www\test\gulpfile.js
 [19:09:05] Starting 'sass'...
 [19:09:06] Finished 'sass' after 699 ms
 [19:09:06] Starting 'default'...
 [19:09:06] Finished 'default' after 11 ms

```
**(B)** shows me an error "Failed to load external module babel-register", starts, opens localhost:8000 and gives me an error every time I change and save assets\scss\_setting.scss 
 
```
 [19:20:10] Starting 'watch'...
 [BS] Access URLs:
  -------------------------------------
        Local: http://localhost:8000
     External: http://192.168.0.14:8000
  -------------------------------------
           UI: http://localhost:3001
  UI External: http://192.168.0.14:3001
  -------------------------------------
 [BS] Serving files from: dist
 [19:20:40] Starting 'sass'...
> Error in plugin 'sass'
 Message:
     src\assets\scss\app.scss
 Error: File to import not found or unreadable: settings
        Parent style sheet: stdin
         on line 3 of stdin
 >> @import 'settings';
    ^
 
 [19:20:40] Finished 'sass' after 19 ms
 [19:20:45] Starting 'sass'...
 Error in plugin 'sass'
 Message:
     src\assets\scss\app.scss
 Error: File to import not found or unreadable: settings
        Parent style sheet: stdin
         on line 3 of stdin
 >> @import 'settings';
    ^

```
Any input would be very appreciated.

**Comment by Larmor27** [comment]
**TL;DR** type your commands with a "$" before them in the git Bash. You can access it via `rightclick into your folder -> Git Bash here`

Alright you windows-newbies out there trying to play with the big iOS-boys like I did, here's an important point:
If you see some shell-command related to node.js written with a dollar sign before it, like `$ do this` or `$ npm install --global foundation-cli` or even `$ foundation new`, be sure to open the Git Bash with `rightclick into your folder -> Git Bash here` and use that as a shell for your commands. Foundation works like charm, thanks for this work guys!
